static/app/views/eventsV2/table/index.tsx

```diff
@@ -96,7 +96,7 @@ class Table extends PureComponent<TableProps, TableState> {
       'discover-frontend-use-events-endpoint'
     );
     const url = shouldUseEvents
-      ? `/organizations/${organization.slug}/events/`
+      ? resolveUrl('organization-events', organization)
```

Comment:
I think this is the interface we're looking for. One change I would make is to have the second argument be a map of key/values that are formatted into the URL so that we can do URLs with project slugs, and group ids etc.